specifications.md

```diff
@@ -0,0 +1,121 @@
+---
+altLangPage: https://design.canada.ca/specifications.html
+date: null
+dateModified: '2024-08-21'
+description: null
+section-title: "Concevoir pour Canada.ca"
+title: "Amélioration continue du contenu Web"
+breadcrumbs: [
+            {
+            "link": "https://conception.canada.ca/",
+            "title": "Concevoir pour Canada.ca"
+          }
+        ]
+---
+
+
+<p>
+ La conception de Canada.ca contribue à offrir une expérience Web harmonisée et digne de confiance aux personnes accédant aux services numériques fédéraux.
+</p>
+ <section>
+  <div class="row">
+   <h2 class="wb-inv">
+    Services and information
+   </h2>
+   <section class="wb-eqht gc-drmt">
+    <div class="col-md-4">
+     <section>
+      <h3 class="h5">
+       <a href="./architecture/utilisation-concept-canadaca.html">
+        Qui doit utiliser la conception de Canada.ca
+       </a>
+      </h3>
+      <p>
+       Champ d’application de la conception de Canada.ca
+      </p>
+     </section>
+    </div>
+    <div class="col-md-4">
+     <section>
+      <h3 class="h5">
+       <a href="./architecture/elements-obligatoires.html">
+        Éléments obligatoires
+       </a>
+      </h3>
+      <p>
+       Couleurs, typographie, en-tête et pied de page, modèles obligatoires
+      </p>
+     </section>
+    </div>
+    <div class="col-md-4">
+     <section>
+      <h3 class="h5">
+       <a href="./architecture/modeles.html">
+        Principes de conception
+       </a>
+      </h3>
+      <p>
+       Concevoir des expériences axées sur les personnes et leurs tâches à l’aide de données probantes
+      </p>
+     </section>
+    </div>
+    <div class="col-md-4">
+     <section>
+      <h3 class="h5">
+       <a href="./specifications/information-trouvabilite.html">
+        Architecture de l’information et trouvabilité
+       </a>
+      </h3>
+      <p>
+       Organiser le contenu, lignes directrices sur les métadonnées, ajouter des données structurées
+      </p>
+     </section>
+    </div>
+    <div class="col-md-4">
+     <section>
+      <h3 class="h5">
+       <a href="./architecture/templates.html">
+        Designing content for Canada.ca
```

Comment:
Change to: Concevoir le contenu pour Canada.ca
https://conception.canada.ca/architecture/modeles.html